conda/recipes/libraft/meta.yaml

```diff
@@ -52,6 +52,9 @@ outputs:
       host:
         - librmm ={{ minor_version }}
         - cudatoolkit {{ cuda_version }}
+      run:
+        - {{ pin_compatible('cudatoolkit', max_pin='x', min_pin='x') }}
```

Comment:
FYI, this is also what the cuda_spec variable is meant for (defined above). If you’re pinning cudatoolkit and don’t have a host pinning to pin_compatible with, you can use cuda_spec.